Level-1/dot/README.md

```diff
@@ -0,0 +1,19 @@
+# rocBLAS-Examples dot
+This example presents independent vectors 'X', 'Y' and a scalar value 'Result' transferred to the GPU device and calling the rocBLAS dot function.Then, The rocBLAS dot function computes the dot product of vectors 'X' and 'Y' and stores the output in 'Result'. Finally, the gold-standard value 'goldResult' (computed in CPU) along with the Result 'hResult' (Computed in GPU) are displayed for comparison.

```

Comment:
Very minor, but looks like there's a space missing: "...calling the rocBLAS dot function.**T**hen, the..."

Reply:
Thanks for the comment. I will make the appropriate changes.